bin/dartdoc.dart

```diff
@@ -53,7 +53,18 @@ void main(List<String> arguments) {
   List<String> excludeLibraries =
       args['exclude'] == null ? [] : args['exclude'].split(',');
   String url = args['hosted-url'];
-  String footer = _resolveTildePath(args['footer']);
+  String footerFilePath = _resolveTildePath(args['footer']);
```

Comment:
Should we write a single function that resolves the Tilde and prints the warning if the file doesn't exist? Looks like two copies of basically the same logic.